maat-court-data-api/src/main/java/gov/uk/courtdata/dces/response/ConcorContributionResponse.java

```diff
@@ -0,0 +1,13 @@
+package gov.uk.courtdata.dces.response;
+
+import lombok.*;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
This class can made immutable as no `set` methods are used and all the fields are immutable (`Integer` and `String`)
You can use the Lombok annotation `@Value` to do this. e.g.
```
@Builder
@Value
public class ConcorContributionResponse {
    Integer concorContributionId;

    @ToString.Exclude
    String xmlContent;
}
```